checker-util/src/test/java/org/checkerframework/checker/regex/util/RegexUtilTest.java

```diff
@@ -23,6 +26,10 @@ public void test_isRegex_and_asRegex() {
         RegexUtil.asRegex(s1, 1);
         Assert.assertFalse(RegexUtil.isRegex(s1, 2));
         Assert.assertThrows(Error.class, () -> RegexUtil.asRegex(s1, 2));
+        Assert.assertTrue(RegexUtil.isRegex(s1, 1, 1));
+        Assert.assertFalse(RegexUtil.isRegex(s1, 1, 1, 2));
```

Comment:
There are no positive tests (that return `true`) of isRegex with more than 3 arguments.  Please add some.  Also add tests where the provided set of non-null regexes is a subset of what it could be, but the result is still true.